velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -15,178 +15,1541 @@
  */
 #include <fmt/format.h>
 
+#include "velox/dwio/dwrf/test/utils/BatchMaker.h"
```

Comment:
BatchMaker is deprecated. Let's use VectorMaker instead. It is available via the base class and base class also has helper methods that makes it easier to use it.

Reply:
Thanks and will make the change in the followup test refactor PR.

Reply:
Actually, I don't see where this header is used. Is it? Would it be possible to first refactor the test in a separate PR? This would avoid a lot of churn in the code and make it easier to review.